lib/MusicBrainz/Server/Controller/CDTOC.pm

```diff
@@ -300,6 +300,7 @@ sub _attach_list {
 
             $c->stash(
                 template => 'cdtoc/attach_filter_release.tt',
+                type => 'add',
```

Comment:
This PR looks good except I usually prefer to use descriptive names for stuff in the stash, since they're basically globals and it makes it easier to grep for all the places they're referenced. Maybe something like `cdtoc_action` here?
```suggestion
                cdtoc_action => 'add',
```